product-service/functions/getProductById.ts

```diff
@@ -11,8 +11,10 @@ const baseResponse = {
 export const getProductById: APIGatewayProxyHandler = async (event, _context) => {
```

Comment:
the same. Also add types

Reply:
I think no need to add types, types are inferred
![image](https://user-images.githubusercontent.com/7376989/98699598-c3130b00-2390-11eb-8306-631bfbaa4c9c.png)

Reply:
First of all using types makes code easier to read and helps to understand how your project is built. It is just no need to open your code in IDE